src/pages/RoversPage.scss

```diff
@@ -0,0 +1,15 @@
+@use "../styles/_variables.scss" as *;
+
+h1 {
```

Comment:
Here you are styling every `h1` on the whole site, please add a specific class name